doc/debug-sections.txt

```diff
@@ -7,6 +7,9 @@
  */
 
 
+section --    External DISKD process implementation.
+section --    Refcount allocator
+section --    Unlink Daemon
```

Comment:
We should probably remove the source code comments that inject these `--` entries, but polishing of individual section comments should stay outside this PR scope IMO.